src/platform/site-wide/mega-menu/data.json

```diff
@@ -288,11 +288,15 @@
                         "links": [
                             {
                                 "text": "Check Your Claim or Appeal Status",
-                                "href": "#"
+                                "href": "/claim-or-appeal-status/"
```

Comment:
I don't think this page exists yet, but the link checker won't scan this menu, so we're safe here.